baybe/utils/conversion.py

```diff
@@ -42,6 +42,18 @@ def nonstring_to_tuple(x: Sequence[_T], self: type, field: Attribute) -> tuple[_
     return tuple(x)
 
 
+def normalize_str_sequence(
```

Comment:
The newly introduced name is misleading and the introduced type hint is false, because:
you introduced this function when we enabled Booleans for categorical parameters. Both name and type hint suggest that the incoming object is already a sequence of strings, which is not the case – it's more generally a sequence of "convertible to string", and right now we use it for str/bool.